assets/airports/gcrr.json

```diff
@@ -23,297 +23,526 @@
+            "height": "28000ft",
+            "coordinates": [
+                ["N27d20m00.00", "W015d50m07.00"],
+                ["N26d49m23.00", "W016d09m17.00"],
+                ["N26d43m09.00", "W016d07m40.00"],
+                ["N26d35m01.00", "W015d42m01.00"],
+                ["N27d19m55.00", "W015d36m45.00"],
+                ["N27d20m00.00", "W015d50m07.00"]
+            ]
+        },
+        {
+            "name": "GCD51",
+            "height": "4000ft",
+            "coordinates": [
+                ["N27d55m50.00", "W015d21m44.00"],
+                ["N27d59m44.00", "W015d16m40.00"],
+                ["N27d52m13.00", "W015d16m20.00"],
+                ["N27d55m50.00", "W015d21m44.00"]
+            ]
+        },
+        {
+            "name": "GCD53",
+            "height": "32000ft",
             "coordinates": [
-                ["N28.9533333", "W13.8266833"],
-                ["N28.9667000", "W13.8266833"],
-                ["N28.9667000", "W13.7933167"],
-                ["N28.9533333", "W13.7933167"]
+                ["N28d10m08.00", "W015d24m22.00"],
+                ["N28d16m39.00", "W015d19m27.00"],
+                ["N28d16m39.00", "W015d31m38.00"],
+                ["N28d10m08.00", "W015d24m22.00"]
             ]
         },
         {
-            "name": "D-3",
+            "name": "GCD54",
+            "height": "4000ft",
+            "coordinates": [
+                ["N28d10m08.00", "W015d25m10.00"],
+                ["N28d12m18.00", "W015d19m06.00"],
+                ["N28d05m30.00", "W015d20m08.00"],
+                ["N28d09m39.00", "W015d25m10.00"],
+                ["N28d10m08.00", "W015d25m10.00"]
+            ]
+        },
+        {
+            "name": "GCD75",
             "height": "6500ft",
             "coordinates": [
-                ["N28.3472500", "W14.1555667"],
-                ["N28.2555833", "W14.1555667"],
-                ["N28.2389000", "W14.2166833"],
-                ["N28.3472500", "W14.2166833"]
+                ["N28d09m30.00", "W015d24m00.00"],
+                ["N28d09m30.00", "W015d26m00.00"],
+                ["N28d11m00.00", "W015d26m00.00"],
+                ["N28d11m00.00", "W015d24m00.00"],
+                ["N28d09m30.00", "W015d24m00.00"]
             ]
         },
         {
-            "name": "R-150",
-            "height": "3500ft",
+            "name": "GCD78",
+            "height": "12000ft",
             "coordinates": [
-                ["N28.9764167", "W13.7683333"],
-                ["N28.9861167", "W13.8400167"],
-                ["N29.0400167", "W13.8189667"],
-                ["N29.0650000", "W13.7775333"],
-                ["N29.0011333", "W13.7064167"]
+                ["N27d46m00.00", "W015d33m00.00"],
+                ["N27d46m00.00", "W015d36m00.00"],
+                ["N27d43m00.00", "W015d36m00.00"],
+                ["N27d43m00.00", "W015d33m00.00"],
+                ["N27d46m00.00", "W015d33m00.00"]
             ]
         }
     ],
     "fixes": {
```

Comment:
There's a few missing fixes, if you wouldn't mind adding those in to clear the warnings. :smiley: 

![image](https://cloud.githubusercontent.com/assets/5103735/26287989/c5fcbfca-3e54-11e7-8517-5bad34d3bef3.png)